generators/kubernetes-helm/templates/csvc/values.yml.ejs

```diff
@@ -0,0 +1,2325 @@
+<%_ if (useKafka) { _%>
+kafka:
+  enabled: true
+#  imageTag: <%= DOCKER_KAFKA %>
+  replicas: 1
+  zookeeper:
+    enabled: true
+<%_ } _%>
+<%_ if (monitoring === 'elk') { _%>
+elk:
+  enabled: true
+elasticsearch:
+  image:
+    repository: "<%= DOCKER_JHIPSTER_ELASTICSEARCH.split(":")[0] %>"
+    tag: "<%= DOCKER_JHIPSTER_ELASTICSEARCH.split(":")[1] %>"
+  cluster:
+    env:
+      MINIMUM_MASTER_NODES: 2
+  master:
+    name: master
+    replicas: 2
+  data:
+    name: data
+    replicas: 1
+  client:
+    name: client
+    replicas: 1
+<%_ } _%>
+<%_ if (monitoring === 'prometheus') { _%>
+prometheus:
+  enabled: true
+  server:
+    name: jhipster-prometheus
+    service:
+      servicePort: 9090
+  nodeExporter:
+    enabled: false
+grafana:
+  ingress:
+    enabled: true
+    hosts:
+    - jhipster-grafana.<%= ingressDomain %>
+  datasources:
+    datasources.yaml:
+      apiVersion: 1
+      datasources:
+      - name: prometheus
+        type: prometheus
+        url: http://jhipster-prometheus:9090
+        access: proxy
+        isDefault: true
+        basicAuth: false
+  dashboardProviders:
+    dashboardproviders.yaml:
+      apiVersion: 1
+      providers:
+      - name: 'prometheus'
+        orgId: 1
+        folder: ''
+        type: file
+        disableDeletion: false
+        editable: true
+        options:
+          path: /var/lib/grafana/dashboards/prometheus
+  dashboards:
+    prometheus:
+      prometheus-dashboard:
+        json: |-
+            {
```

Comment:
It's cool that you have set up a dashboard like that but unfortunately we are migrating to micrometer soon so all of those will be obsolete. And  we are going to develop custom dashboards with @clement26695 that we will properly package in the jhipster-console sub-project.

Reply:
Yes, was done a while ago :). Based on the progress, let's update later on.